tests/defaults.yml

```diff
@@ -0,0 +1,2 @@
+applications:
+  - cortex-tenant-ns-label
```

Comment:
You don't have to specify the component in `applications` in the test input unless your code interacts with `inv.applications` for some reason,